lib/karma-electron-preprocessor.js

```diff
@@ -38,6 +38,7 @@ function createElectronPreprocessor(karmaBasePath, karmaClientConfig, karmaCusto
     var sourceMap = convertSourceMap.fromSource(content);
     var output = template({
       __filenameOverride: karmaClientConfig.__filenameOverride,
+      __injectionVariable: karmaClientConfig.__injectionVariable || '__ELECTRON_INJECTED__',
```

Comment:
Just in the very unlikely case that this variable conflicts with downstream... who knows.